### DISCUSSION TITLE
[Bug]: Docker-Based installation with 2024.2 fails | ext-ampq missing in Docker image

### DISCUSSION THREAD

**Discussion by beheist** [pimcore/pimcore#17332] [Q&A / Support]
### Pimcore version

2024.2

### Steps to reproduce

Run the command for docker-based installation as documented in https://pimcore.com/docs/platform/2023.3/Pimcore/Getting_Started/Installation/Docker_Based_Installation.

### Actual Behavior

Installation fails with the following message: 
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - symfony/amqp-messenger[v6.2.0, ..., v6.4.8] require ext-amqp * -> it is missing from your system. Install or enable PHP's amqp extension.
    - Root composer.json requires symfony/amqp-messenger ^6.2 -> satisfiable by symfony/amqp-messenger[v6.2.0, ..., v6.4.8].

I assume this is because ext-ampq is not installed in the default Pimcore Docker image.

### Expected Behavior

Installations runs without problems.

**Comment by jdreesen** [comment]
> I assume this is because ext-ampq is not installed in the default Pimcore Docker image.

It is installed, see https://github.com/pimcore/docker/pull/152, https://github.com/pimcore/docker/pull/159 and https://github.com/pimcore/docker/releases/tag/v3.3

Are you sure you're using the latest images (`v3.3`)?

**Comment by beheist** [comment]
Well, I'm using the ones that docker pulls when I run the install command that is provided in the docs:

```
docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.2-latest composer create-project pimcore/demo my-project
```

Could it be that docker is using a locally cached one instead of checking for newer ones on docker hub?

**Comment by beheist** [comment]
So, using this command, the 2024.2 installation works:
```
docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.2-v3.3 composer create-project pimcore/skeleton my-project
```

Would it not make more sense to have the appropriate docker image version in the install command for each respective version of Pimcore, so it is made sure you get the correct image version?

**Comment by fashxp** [comment]
it should work with 8.2-latest too. maybe you need to do a `docker pull` first?
